eos-phone-home

```diff
@@ -309,7 +313,8 @@ class PhoneHome(object):
             log.info("Getting variable: %s", name)
             var = getattr(self, '_get_' + name)()
 
-            if var is None or var == '':
+            if var in self.MANDATORY_STRINGS and \
```

Comment:
This will always fail, unless the DMI data reports a vendor of `"vendor"` – hey actually not that unlikely! You mean `name in self.MANDATORY_STRINGS`.

Reply:
Clucking bell! That would've been... undesirable. Thank you. 🍻